ecephys_spike_sorting/scripts/create_input_json.py

```diff
@@ -57,32 +66,39 @@ def createInputJson(output_file,
                     ks_LTseed = 1,
                     ks_templateRadius_um = 163,
                     c_Waves_snr_um = 160,
-                    qm_isi_thresh = 1.5/1000                    
+                    qm_isi_thresh = 1.5/1000
                     ):
 
     # hard coded paths to code on your computer and system
-    ecephys_directory = r'D:\ecephys_fork\ecephys_spike_sorting\ecephys_spike_sorting'
-    
+    # ecephys_directory = r'D:\ecephys_fork\ecephys_spike_sorting\ecephys_spike_sorting'
+    ecephys_directory = ecephys_directory
+
     # location of kilosor respository and kilosort version
 
-    kilosort_repository = r'C:\Users\labadmin\Documents\jic\KS3_fork\Kilosort2'
-    KS2ver = '2.0'      # must equal '3.0', '2.5' or '2.0', and match the kiilosort_repository
-    
+    # kilosort_repository = r'C:\Users\labadmin\Documents\jic\KS3_fork\Kilosort2'
+    # KS2ver = '2.0'      # must equal '3.0', '2.5' or '2.0', and match the kiilosort_repository
+    kilosort_repository = kilosort_repository
+    KS2ver = KS2ver
+
     # KS 3.0 does not yet output pcs.
     if KS2ver == '3.0':
         include_pcs = False  # set to false for KS2ver = '3.0'
     else:
         include_pcs = True
     
-    npy_matlab_repository = r'C:\Users\labadmin\Documents\jic\npy-matlab-master'
-    catGTPath = r'C:\Users\labadmin\Documents\jic\CatGT-win'
-    tPrime_path=r'C:\Users\labadmin\Documents\jic\TPrime-win'
-    cWaves_path=r'C:\Users\labadmin\Documents\jic\C_Waves-win'
-    
+    #npy_matlab_repository = r'C:\Users\labadmin\Documents\jic\npy-matlab-master'
+    #catGTPath = r'C:\Users\labadmin\Documents\jic\CatGT-win'
+    #tPrime_path=r'C:\Users\labadmin\Documents\jic\TPrime-win'
+    #cWaves_path=r'C:\Users\labadmin\Documents\jic\C_Waves-win'
+    npy_matlab_repository = npy_matlab_repository
```

Comment:
What do these assignments do?

Reply:
The original code's path assignments were hardcoded, but I need them to be passed through the arguments. Those are just temp reminders for myself while I was coding, I'll remove them and add comments instead.

Reply:
https://github.com/vathes/ecephys_spike_sorting/pull/2/commits/6dbe139c9eb62b8ed8132b88bbd40f446d535e63